### DISCUSSION TITLE
Problem in installing atom

### DISCUSSION THREAD

**Discussion by aaelmeli** [idaholab/moose#18653] [Q&A General] [answered]
Hi
I downloaded atom for windows from the link provided in MOOS's [link](https://mooseframework.inl.gov/help/development/Atom_Editor.html). I installed it using the downloaded file and I could open it. However, I do not know how to do this command `select _Atom->Install Shell Commands_` shown in the link above. 
And when I do atom in the terminal, I am getting this message 

> (moose) aaelmeli@CCEE-DT-284:~/projects$ atom
> '\\wsl$\Ubuntu-20.04\home\aaelmeli\projects'
> CMD.EXE was started with the above path as the current directory.
> UNC paths are not supported.  Defaulting to Windows directory.

so, can you please help me to fix this?
thank you.
Abdo.

**Comment by dschwen** [comment, marked as answer]
Launch Atom from the Windows Start menu

**Reply by aaelmeli** [reply]
Thank you, dschwen, for the reply. 
In this case, I think I have already done that. However, I am still getting the above error message (in the original post) when trying to launch it from the terminal using `atom` . I think it has to be something with the path.

**Reply by hugary1995** [reply]
I remember in windows you can open a folder, then right click, and the context menu will have an option to let you open the current directory in atom. Can you try that?

The tutorial on the moose website is specific to MacOS I believe, while it is similar in Linux. But not for windows.

**Reply by dschwen** [reply]
> In this case, I think I have already done that. 

Good

> However, I am still getting the above error message (in the original post) when trying to launch it from the terminal using `atom`.

Then _don't_ do that. :-D
